File: 2125-number-of-laser-beams-in-a-bank/2125-number-of-laser-beams-in-a-bank.cpp
class Solution {
public:
    int numberOfBeams(vector<string>& bank) {
        int prevCamCount = 0;
        int cameraCount = 0;
        int beams = 0;
        for(int i = 0; i < bank.size(); i++){
            for(int j = 0; j < bank[0].size(); j++){
                if(bank[i][j] == '1') cameraCount++;
            }
            cout<<cameraCount;
            if(cameraCount){
                beams = beams+prevCamCount*cameraCount;
                prevCamCount = cameraCount;
            }
            cameraCount = 0;
        }
        return beams;
    }
};

File: 2125-number-of-laser-beams-in-a-bank/2125-number-of-laser-beams-in-a-bank.py
class Solution:
    def numberOfBeams(self, bank: List[str]) -> int:
        prevCamCount = 0
        cameraCount = 0
        beams = 0
        for i in range(len(bank)):
            for j in range(len(bank[0])):
                if bank[i][j] == '1' : cameraCount += 1
            if cameraCount : 
                beams += prevCamCount*cameraCount
                prevCamCount = cameraCount
            cameraCount = 0
        return beams